sites/all/themes/wille/assets/sass/component/_article.scss

```diff
@@ -36,6 +36,10 @@ $z-index-cover-base: 1;
       margin-bottom: 32px;
     }
 
+    li {
+      line-height: 155%;
```

Comment:
The preferred way to set line-height is using a unitless number. F.x. 1.5 - this to avoid unexpected results due to inheritance.

https://developer.mozilla.org/en-US/docs/Web/CSS/line-height

Reply:
`line-height: 155%;` was lifted from [_article.scss#L34](https://github.com/eReolen/base/blob/develop/sites/all/themes/wille/assets/sass/component/_article.scss#L34), but https://github.com/eReolen/base/pull/256/commits/324ef699383e373bc3360e97c57c77abfb19bd30 cleans this up a bit.